tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -23,7 +23,9 @@ namespace tensorflow {
 namespace {
 
 // In case of failure, every call will be retried kMaxAttempts-1 times.
-constexpr int kMaxAttempts = 4;
+constexpr int kMaxAttempts = 3;
```

Comment:
`attempt++` happens after the check now. Hence this has been changed from 4 to 3. I need to update the comment however has now it retries exactly `kMaxAttempts` times (if the comparison check is changed to a `>=` or a `==`)